src/components/Table/style.module.scss

```diff
@@ -24,5 +24,10 @@
 		line-height: 22px;
 		color: $second-main-color;
 	}
-}
 
+	&__list {
+		overflow-y: scroll;
+		height: 586px;
```

Comment:
page - 100vh

height of header = 20vh|300px, list - calc(100vh - header.height)